JetAnalyzers/bin/jet_synchfit_x.cc

```diff
@@ -25,194 +26,512 @@
+        rho_residual_graphs[irefpt]->Write();
+      }
+
+      fout->mkdir(Form("residualgraphs/p_offOverA_EtaVsJetPt/Eta%d",iEta));
+      fout->cd(Form("residualgraphs/p_offOverA_EtaVsJetPt/Eta%d",iEta));
+
+      for (unsigned itnpu = 0; itnpu<ptpu_residual_graphs.size(); itnpu++){
+        if (!ptpu_residual_graphs[itnpu]->GetN()) continue;
+        ptpu_residual_graphs[itnpu]->SetNameTitle(Form("ieta_%d_itnpu_%d_residual",iEta,itnpu+1),Form("ieta_%d_itnpu_%d_residual;p_{T}^{pu};ooa",iEta,itnpu+1));
+        ptpu_residual_graphs[itnpu]->Write();
+      }
+
+    }// iEta
+
+
+    fout->cd("histograms/refpt");
+
+    for (unsigned irefpt = 0; irefpt<refpt_hists.size(); irefpt++) if (refpt_hists[irefpt]->GetEntries()){
+      refpt_hists[irefpt]->SetStats(0);
+      refpt_hists[irefpt]->Write();
+    }
+
+    fout->cd("histograms/tnpu");
+
+    for (unsigned itnpu = 0; itnpu<tnpu_hists.size(); itnpu++) if (tnpu_hists[itnpu]->GetEntries()){
+      tnpu_hists[itnpu]->SetStats(0);
+      tnpu_hists[itnpu]->Write();
+    }
+
+    fout->cd("residual/refpt");
+
+    for (unsigned irefpt = 0; irefpt<refpt_residual_hists.size(); irefpt++) if (refpt_residual_hists[irefpt]->GetEntries()){
+      refpt_residual_hists[irefpt]->SetStats(0);
+      refpt_residual_hists[irefpt]->Write();
+    }
+
+    fout->cd("residual/tnpu");
+
+    for (unsigned itnpu = 0; itnpu<tnpu_residual_hists.size(); itnpu++) if (tnpu_residual_hists[itnpu]->GetEntries()){
+      tnpu_residual_hists[itnpu]->SetStats(0);
+      tnpu_residual_hists[itnpu]->Write();
+    }
+
+    fout->cd("rel_residual/refpt");
+
+    for (unsigned irefpt = 0; irefpt<refpt_residual_hists.size(); irefpt++) if (refpt_residual_hists[irefpt]->GetEntries()){
+      refpt_residual_hists[irefpt]->SetStats(0);
+      refpt_residual_hists[irefpt]->Write();
+    }
+
+    fout->cd("rel_residual/tnpu");
+
+    for (unsigned itnpu = 0; itnpu<tnpu_residual_hists.size(); itnpu++) if (tnpu_residual_hists[itnpu]->GetEntries()){
+      tnpu_relative_residual_hists[itnpu]->SetStats(0);
+      tnpu_relative_residual_hists[itnpu]->Write();
+    }
 
+    fout->cd("/");
+
+    h_rms->Write();
+    h_mean->Write();
+    h_eff_entries->Write();
+
+    cout<<endl<<endl;
+} // fitClosurePlots
 
 //===========================================================================
 // This method tries to obtain all the relevant TProfiles from the inputFilename
 // It returns true if successfull
-bool getInputProfiles(TString inputFilename, TProfile3D *& prof, 
-                      TProfile3D *& profPt, 
-                      TProfile3D *& profRho, bool useNPU){
-   
-   // Open the input file
-   TFile *fin= new TFile(inputFilename);
-   if(!fin->IsOpen()) {
-      cout << "ERROR jet_synchfit_xx::getInputProfiles() could not open file " 
-           <<inputFilename<< endl;
+bool getInputProfiles(TString inputFilename, TProfile3D *& prof,
```

Comment:
No white space changes unless necessary and globally matched.

Reply:
I'll fix that here as well, thank you.